fe1-web/src/features/rollCall/components/RollCallOpen.tsx

```diff
@@ -71,20 +80,13 @@ const RollCallOpen = ({
         screen: STRINGS.events_open_roll_call,
         params: {
           rollCallId: rollCall.id.toString(),
-          attendeePopTokens: (scannedPopTokens || []).map((e) => e.valueOf()),
+          attendeePopTokens: allAttendees.map((e) => e.valueOf()),
         },
       },
     });
-  }, [toast, navigation, rollCall, scannedPopTokens]);
+  }, [toast, navigation, rollCall, allAttendees]);
 
   const onCloseRollCall = useCallback(async () => {
-    // get the public key as strings from the existing rollcall
-    const previousAttendees = rollCall.attendees || [];
-    // add the create a set of all attendees (takes care of deduplication)
-    const allAttendees = new Set([...previousAttendees, ...(scannedPopTokens || [])]);
-    // convert it back to a list
-    const attendeesList = [...allAttendees];
```

Comment:
how do you ensure deduplication without this logic? 🤔

Reply:
Because we control the scanner. `ScannedPopTokens ` is non-empty, only if we come back from the scanner, which already has the scanned attendees. Then the scanner takes to only add a new token if it is not already in the list.

Reply:
I think this is now handled by always passing around the full list of tokens and the scanner preventing to scan any duplicates, right @MeKHell? 😅

Reply:
Yes